.drone/drone.yml

```diff
@@ -190,6 +190,38 @@ depends_on:
 - docker-amd64
 - docker-arm64
 
+---
+kind: pipeline
+name: cd-to-dev-env
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: image-tag
+  image: alpine/git
+  commands:
+  - apk --update --no-cache add bash
+  - git fetch origin --tags
+  - echo $(./tools/image-tag) > .tags
+
+- image: us.gcr.io/kubernetes-dev/drone/plugins/updater
+  settings:
+    config_json: "{\n  \"destination_branch\": \"master\",\n  \"pull_request_branch_prefix\": \"cd-tempo-dev\",\n  \"pull_request_enabled\": false,\n  \"pull_request_team_reviewers\": [\n    \"tempo-squad\"\n  ],\n  \"repo_name\": \"deployment_tools\",\n  \"update_jsonnet_attribute_configs\": [\n    {\n      \"file_path\": \"ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet\",\n      \"jsonnet_key\": tempo,\n      \"jsonnet_value_file\": \".tags\"\n    }\n    {\n      \"file_path\": \"ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet\",\n      \"jsonnet_key\": tempo-query,\n      \"jsonnet_value_file\": \".tags\"\n    }\n    {\n      \"file_path\": \"ksonnet/environments/tempo/dev-us-central-0.tempo-dev-01/images.libsonnet\",\n      \"jsonnet_key\": tempo-vulture,\n      \"jsonnet_value_file\": \".tags\"\n    }\n  ]\n}\n"
```

Comment:
I wish this could be formatted better. Not sure if there's a way to do that.

Reply:
This might be possible with `std.manifestJsonEx`?
https://jsonnet.org/ref/stdlib.html#:~:text=std.manifestJsonEx(value%2C%20indent%2C%20newline%2C%20key_val_sep)

Reply:
You can, check https://github.com/grafana/agent/blob/a9bc23c74f3727399e4f720abbbe9a3b5fc8b973/.drone/drone.yml#L151-L168 as an example.

Reply:
Ah, nevermind. You are actually generating this from jsonnet :)

Reply:
Hehe yes, formatting didn't work, but want to get this in, so will merge :)